sphinxcontrib/jsonschema.py

```diff
@@ -144,28 +145,26 @@ def row(self, prop, tbody):
                 if ref:
                     # just use the name at the end of the ref
                     ref = ref.split('/')[-1]
-                    reference = nodes.reference('', '',
-                        nodes.Text(ref),
-                        internal=False,
-                        refuri='#'+ref.lower(), anchorname='')
+                    reference = nodes.reference('', '', nodes.Text(ref), internal=False, refuri='#' + ref.lower(),
+                                                anchorname='')
                     cell += nodes.paragraph('', nodes.Text('\n\nSee '), reference)
                 if prop.deprecated:
-                    cell += nodes.paragraph('',
-                        nodes.Text('This property was deprecated in version {}'.format(prop.deprecated['deprecatedVersion'])),
-                    )
-                    cell += nodes.paragraph('',
-                        nodes.Text(prop.deprecated['description']),
-                    )
+                    cell += nodes.paragraph('', nodes.Text('This property was deprecated in version {}'
+                                                           .format(prop.deprecated['deprecatedVersion'])))
+                    cell += nodes.paragraph('', nodes.Text(prop.deprecated['description']))
             row += cell
         tbody += row
 
     def cell(self, text, morecols=0, source=None):
         entry = nodes.entry(morecols=morecols)
         if not isinstance(text, string_types):
             text = str(text)
-        # Regex to replace markdown links for reStructuredText ones
-        text = re.sub(r'\[([^\]]+)\]\(([^\)]+)\)', r'`\1 <\2>`__', text)
-        viewlist = ViewList(text.split('\n'), source=source)
+
+        parser = CommonMarkParser()
+        new_doc = new_document(None)
+        parser.parse(text, new_doc)
+
+        viewlist = ViewList([child.astext() for child in new_doc.children[:]], source=source)
```

Comment:
I think `astext` removes the links!
Here's what I think this should look like: https://github.com/OpenDataServices/sphinxcontrib-jsonschema/commit/01c4d26f782a9bf212cc14110d14a943a0580e75

(We don't need the `nested_parse`, call as that's for paring RST).